domains/red.json

```diff
@@ -6,6 +6,6 @@
     "email": "hello@standred.dev"
   },
   "record": {
-    "web": "standred.dev"
+    "web": "www.standred.dev"
```

Comment:
Please change this to `"URL": "https://www.strandred.dev"` if you want it to redirect.